src/routes/RemediationDetails.js

```diff
@@ -37,7 +38,8 @@ class RemediationDetails extends Component {
     constructor (props) {
         super(props);
         this.state = {
-            autoReboot: true
+            autoReboot: true,
+            isUserEntitled: true
```

Comment:
Can we make the default value `undefined`? That way neither the banner is shown nor the Execute button is shown before we figure out if the user is entitled. Once the call to `getUser()` returns we either show the button or the banner. (Trying to prevent the button showing up for a moment before it is replaced by the banner).